route/test/test_route_generator.cpp

```diff
@@ -25,36 +25,83 @@
-    //    lanelet::projection::LocalFrameProjector("+proj=tmerc +lat_0=39.46636844371259 +lon_0=-76.16919523566943 +k=1 +x_0=0 +y_0=0 +units=m +vunits=m"));
-    lanelet::LaneletMapPtr map = lanelet::load("/home/qswawrq/Desktop/ATEF_pretty.osm", lanelet::Origin({0, 0}));
+
+    int projector_type = 0;
+    std::string target_frame;
+    lanelet::ErrorMessages load_errors;
+
+    // If the output is an error about the geoReference field in the osm file, here is a correct example. If the lat/lon coordinates are already correct, simply add:
+    // <geoReference>+proj=tmerc +lat_0=0 +lon_0=0 +k=1 +x_0=0 +y_0=0 +datum=WGS84 +units=m +geoidgrids=egm96_15.gtx +vunits=m +no_defs</geoReference>
+
+    // File location of osm file
+    std::string file = "/workspaces/carma_ws/carma/src/Town02_pretty.osm";
+    // Starting and ending lanelet IDs. It's easiest to grab these from JOSM
+    lanelet::Id start_id = 111;
+    lanelet::Id end_id = 144;
+
+    // The parsing in this file was copied from https://github.com/usdot-fhwa-stol/carma-platform/blob/develop/carma_wm_ctrl/test/MapToolsTest.cpp
+    lanelet::io_handlers::AutowareOsmParser::parseMapParams(file, &projector_type, &target_frame);
+    lanelet::projection::LocalFrameProjector local_projector(target_frame.c_str());
+    lanelet::LaneletMapPtr map = lanelet::load(file, local_projector, &load_errors);
+
+    // Grabs lanelet elements from the start and end IDs. Fails the unit test if there is no lanelet with the matching ID
+    lanelet::Lanelet start_lanelet;
+    lanelet::Lanelet end_lanelet;
+    try {
+        start_lanelet = map->laneletLayer.get(start_id);
+    }
+    catch (const lanelet::NoSuchPrimitiveError& e) {
+        FAIL() << "The specified starting lanelet Id of " << start_id << " does not exist in the provided map.";
+    }
+    try {
+        end_lanelet = map->laneletLayer.get(end_id);
+    }
+    catch (const lanelet::NoSuchPrimitiveError& e) {
+        FAIL() << "The specified ending lanelet Id of " << end_id << " does not exist in the provided map.";
+    }
+
     lanelet::LaneletMapConstPtr const_map(map);
-    lanelet::BasicPoint2d start(-1056.75, -70.6);
-    std::vector<lanelet::BasicPoint2d> via;
-    // via.emplace_back(lanelet::BasicPoint2d(-158.0, 533.5));
-    //via.emplace_back(lanelet::BasicPoint2d(-17.5, 322.0));
-    lanelet::BasicPoint2d end(1052.35, 771.24);
     lanelet::traffic_rules::TrafficRulesUPtr traffic_rules = lanelet::traffic_rules::TrafficRulesFactory::create(lanelet::Locations::Germany, lanelet::Participants::VehicleCar);
     lanelet::routing::RoutingGraphUPtr map_graph = lanelet::routing::RoutingGraph::build(*map, *traffic_rules);
     // Output graph for debugging
-    map_graph->exportGraphViz("/home/qswawrq/Desktop/routing.txt");
-    auto route = worker.routing(start, via, end, const_map, std::move(map_graph));
+    map_graph->exportGraphViz("/workspaces/carma_ws/carma/src/routing.txt");
```

Comment:
Same here with the filepath, I'd suggest making this a relative path so that it works no matter the home folder/user.